.github/CODEOWNERS

```diff
@@ -243,6 +243,8 @@
 /core/templates/services/page-title.service*.ts @ankita240796
 /core/templates/services/extension-tag-assembler.service*.ts @ankita240796
 /core/templates/services/nested-directives*.ts @ankita240796
+/core/templates/services/KeyboardShortcutService.ts @jimbyo
```

Comment:
Hi @jimbyo, one question -- would you still like to contribute to the Oppia project after GSoC?

If so, I think this is OK, but if not, perhaps we should assign it to someone else.

Also, shouldn't the convention be keyboard-shortcut.service.ts?